test/manifest.js

```diff
@@ -44,15 +44,13 @@ module.exports = [
     // Not supported on chromium, needs source maps.
     // https://github.com/RecordReplay/chromium/issues/5
     targets: ["gecko"],
-    pending: true,
   },
   {
-    // This test is pretty darn broken. The sources are different when recording
-    // the example in CI vs. locally
     example: "doc_navigate.html",
     script: "breakpoints-07.js",
-    pending: true,
     targets: ["gecko", "chromium"],
+    // Disabled because the test contains a navigation that playwright doesn't handle well
+    pending: true,
```

Comment:
I saw this being named `disabled` or `skip` in testing frameworks. Had to look it up what it was `pending` for. 🙃  (just a suggestion)